### DISCUSSION TITLE
Port to Obsidian

### DISCUSSION THREAD

**Discussion by RobyOne77** [daltonmenezes/aura-theme#112] [Port request]
Can port to Obsidian (https://obsidian.md)

Thanks in advance!

**Comment by daltonmenezes** [comment]
Hi @RobyOne77 , are you interested in making this port for Obsidian?

**Comment by possibly-not** [comment]
I've started work on a port!
Compiled theme here:  https://github.com/Possibly-Matt/obsidian-aura-theme

I will submit a pull-request to put the source code in this repo once I get the theme added to the Obsidian community themes section. That way the installation instructions can just be "install via searching on the community themes"

Currently it's not a great process to have to manually copy over the theme from my aura-theme fork into the theme repo, but in order to submit the theme it needs to be in its own repo.  Not sure if using a git submodule would work?

**Reply by possibly-not** [reply]
My personal theme repo got added to the obsidian one!  https://github.com/obsidianmd/obsidian-releases/pull/2259
![image](https://github.com/daltonmenezes/aura-theme/assets/12588174/9fbce1a1-e725-42ba-a7a7-915bc9eb9e00)
You can now install the Aura Dark theme by searching for it in the themes searcher. 
When I have some time I'll pop-up a pull request from https://github.com/Possibly-Matt/aura-theme
